gobblin-docker/gobblin-wikipedia/ubuntu-gobblin-0.7.0/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gobblin/gobblin-distributions:ubuntu-gobblin-0.7.0
+
+ENV GOBBLIN_WORK_DIR /home/gobblin/work-dir
+
+RUN echo "Downloading Gobblin Wikipedia Configuration Files"
+RUN mkdir -p /etc/opt/gobblin/job-conf
+
+# Downloading wikipedia.pull
+RUN curl -L --progress-bar https://raw.githubusercontent.com/linkedin/gobblin/master/gobblin-example/src/main/resources/wikipedia.pull -o /etc/opt/gobblin/job-conf/wikipedia.pull
```

Comment:
It's not in the distribution unfortunately, although it probably should be. Created #1316 to track this.

Updated to use the release tag. I can't use it for the gobblin-cli.properties file unfortunately since it didn't exist in those releases